### PR TITLE
Trophy of Battle - Secondary Transmitter Drone Core

### PR DESCRIPTION
**Pull request by RustingWithYou**
Adds the Secondary Transmitter Drone Core from today's event to the Horizon for science to poke. It spawns in one of the xenoarchaeology cells, which has been modified to use general science access

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

File: code/game/objects/structures/hivebot_head.dm [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/obj/structure/hivebot_head
+	name = "Secondary Transmitter Drone Core"
```
Comment (SleepyGemmy) [inline code/game/objects/structures/hivebot_head.dm:2]:
```suggestion
	name = "\improper Secondary Transmitter Drone core"
```

File: code/game/objects/structures/hivebot_head.dm [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+/obj/structure/hivebot_head/attack_hand(mob/user)
+	. = ..()
+	user.visible_message(SPAN_NOTICE("\The [user] touches \the [src]."), SPAN_NOTICE("You touch \the [src]."))
+	if(prob(10))
+		var/T = get_turf(src)
+		icon_state = "hivebot_head_active"
+		playsound(src.loc, 'sound/effects/creatures/hivebot/hivebot-bark-005.ogg')
+		to_chat(user, SPAN_WARNING("\The [src] suddenly sparks and lights up, emitting some unintelligible noise!"))
+		spark(T, 3, GLOB.alldirs)
+		sleep(16)
```
Comment (FluffyGhoster) [inline code/game/objects/structures/hivebot_head.dm:23]: No sleeping, use a timer

**Review by FluffyGhoster — CHANGES_REQUESTED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

File: code/game/objects/structures/hivebot_head.dm [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	. = ..()
+	user.visible_message(SPAN_NOTICE("\The [user] touches \the [src]."), SPAN_NOTICE("You touch \the [src]."))
+	if(prob(10))
+		var/T = get_turf(src)
+		icon_state = "hivebot_head_active"
+		playsound(src.loc, 'sound/effects/creatures/hivebot/hivebot-bark-005.ogg')
+		to_chat(user, SPAN_WARNING("\The [src] suddenly sparks and lights up, emitting some unintelligible noise!"))
+		spark(T, 3, GLOB.alldirs)
+		addtimer(CALLBACK(src, PROC_REF(deactivate)), 3 SECONDS)
+
```
Comment (SleepyGemmy) [inline code/game/objects/structures/hivebot_head.dm:24]:
```suggestion
```

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**